server/config.js

```diff
@@ -5,6 +5,15 @@ const cache = new Map()
 const defaultConfig = {
   webpack: null,
   webpackDevMiddleware: null,
+  webpackModuleShouldBeCommonInProduction: function ({totalPages, count}) {
```

Comment:
That's a pretty long function name and it's not so clear, I understood by the name that I'll be able to define if an individual module should be in the common chunks.

Reply:
Hah, fair enough. Technically you could/(should) pass the whole `module` into this method if it were to exist, which would make the method name correct (if long).

In any case I think I'll abort this approach per @timneutkens's comment